portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/Forms.testcase

```diff
@@ -696,13 +696,37 @@ definition {
 	}
 
 	@description = "Verify that a Form can be deleted"
-	@ignore = "Test Stub"
 	@priority = "5"
 	test CanBeDeleted {
 		property forms.functional = "true";
 		property portal.acceptance = "false";
+	
+		ProductMenu.gotoPortlet(
+			category = "Content &amp; Data",
+			portlet = "Forms");
+
+		FormsAdminNavigator.gotoAddForm();
+
+
+		Form.editName(formName = "Form 1");
+
+		Form.addTextField(fieldPositionNumber = "0");
 
-		// TODO LPS-130509 CanBeDeleted pending implementation
+		Form.save();
+
+		ProductMenu.gotoPortlet(
+			category = "Content &amp; Data",
+			portlet = "Forms");
+
+		AssertElementPresent(
+			key_formName = "Form 1",
+			locator1 = "FormsAdmin#FORMS_TABLE_NAME");
+
+		LexiconEntry.gotoEllipsisMenuItemNoError(menuItem = "Delete");
+
+		AssertConfirm(value1 = "Are you sure you want to delete this? It will be deleted immediately.");
+
+		Alert.viewSuccessMessage();
```

Comment:
This looks good, but it would be even better if created inside a new macro, so we could reuse it in other tests - I've checked and we don't have one already created for this.

May I ask you to to create a macro that fills this scope?

A few notes:
- I believe the best place to create it would be on [FormsAdmin.macro ](https://github.com/liferay/liferay-portal/blob/master/portal-web/test/functional/com/liferay/portalweb/macros/FormsAdmin.macro)file
- It would be great if this macro could receive the form name as a parameter

Thank you!

Reply:
@brunofariasdeo requests done!